Makefile

```diff
@@ -18,7 +18,7 @@ JAVA=/usr/bin/java
 CLASSPATH=./classes
 VER=1.1
 REL=5
-ARCH=pp64le
+ARCH=$(shell uname -p)
 PROD=ibm-crassd
 NAME=$(PROD)-$(VER)-$(REL).$(ARCH)
```

Comment:
I suggest adding the RHEL version here as well. The spec files will need to call out the different requirements for RHEL 7 requirements differently due to the change in package naming conventions.

Reply:
Added the rhel version as part of the Release after reading https://docs.fedoraproject.org/en-US/packaging-guidelines/DistTag/ and other similar suggestions. More info in the new commits